site/content/usage/experimental/01-gitops.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "GitOps"
+weight: 10
+url: usage/experimental/gitops-flux
+---
+
+## GitOps
+
+[GitOps][gitops] is a way to do Kubernetes cluster management and application delivery. It works by using Git as a single source
```

Comment:
I think we cannot talk about cluster management yet.

Reply:
I just copied this from https://www.weave.works/technologies/gitops/ 

What do you think it should say instead?

Reply:
Probably best to remove mentions from this paragraph:

> [GitOps][gitops] is a way to do Kubernetes application delivery. It works by using Git as a single source
> of truth for Kubernetes resources. With Git at the center of your delivery pipelines, developers
> can make pull requests to accelerate and simplify application deployments and operations tasks to Kubernetes.